packages/docusaurus/src/server/plugins/init.ts

```diff
@@ -9,8 +9,45 @@ import Module from 'module';
 import {join} from 'path';
 import importFresh from 'import-fresh';
 import {LoadContext, Plugin, PluginConfig} from '@docusaurus/types';
+import {Schema} from 'yup';
+const chalk = require('chalk');
 import {CONFIG_FILE_NAME} from '../../constants';
 
+function printError(error) {
+  console.log(
+    chalk.red(
+      `Validation Errors:${error?.errors?.reduce(
```

Comment:
We should probably not assume that the user will want to use a Joi/Yup schema here. 

He may as well just throw a regular JS error with a message, in such case we should print directly that message.